src/app/routes/api/v3/forms/public-forms.submissions.routes.ts

```diff
@@ -65,3 +65,10 @@ PublicFormsSubmissionsRouter.route(
   limitRate({ max: rateLimitConfig.submissions }),
   EncryptSubmissionController.handleStorageSubmission,
 )
+
+PublicFormsSubmissionsRouter.route(
+  '/:formId([a-fA-F0-9]{24})/submissions/storage/get-s3-presigned-post-data',
+).post(
+  limitRate({ max: rateLimitConfig.submissions }),
+  EncryptSubmissionController.handleGetS3PresignedPostData,
+)
```

Comment:
suggestion: add jsdocs and tests for this in `public-forms.submissions.routes.spec.ts``

Reply:
oh my, I'm missing tests for the `/submissions/storage` route too

Reply:
docs added here: [a42c52e](https://github.com/opengovsg/FormSG/pull/6685/commits/a42c52e7157af3dd0dded81a2574710fd328eb76)

Reply:
Tests for `/submissions/storage/get-s3-presigned-post-data` added in [6922dce](https://github.com/opengovsg/FormSG/pull/6685/commits/6922dceba9ff631b87946ff8c0443beb08521f0c).